<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opcije</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#OptionsDialog {
    background-color: rgb(84, 84, 84);
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="fileHLayout">
     <item>
      <widget class="QLabel" name="FileLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Datoteka: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileLineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseFileButton">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(84, 84, 84);
	border-color: white
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../resources/icons/browse.png</normaloff>../../../resources/icons/browse.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="timeHLayout" stretch="1,0,0,1,1">
     <item>
      <widget class="QLabel" name="FromLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Vremenski interval:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FromLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhTime</set>
       </property>
       <property name="inputMask">
        <string>99:99:99</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DashLabel">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#DashLabel {
	padding-left: 5px;
}
	</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;   -&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ToLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhTime</set>
       </property>
       <property name="inputMask">
        <string>99:99:99</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBox" stretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#CancelButton {
    border-radius: 6px;
    border: 3px solid #9C8ABF;

    font-weight: 600;
    font-size: 14px;
    color: white;
}

QPushButton#CancelButton:hover {
    border-color: #B8A2E1; 
}

QPushButton#CancelButton:pressed {
    background-color: rgb(70, 70, 70);
}</string>
       </property>
       <property name="text">
        <string>Odustani</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OKButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777213</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#OKButton {
    border-radius: 6px;
    border: 3px solid #54a4a6;

    font-weight: 600;
    font-size: 14px;
    color: white;
}

QPushButton#OKButton:hover {
   border-color: #63C2C4;
}

QPushButton#OKButton:pressed {
    background-color: rgb(70, 70, 70)
}

QPushButton#OKButton:disabled {

	color: rgb(153, 153, 153);
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
